cogs/moveusers.py

```diff
@@ -1,13 +1,95 @@
 import discord
 from discord.ext import commands
+from difflib import SequenceMatcher
+from constantvariables import DIFFERENT_NAME_CHANNELS
 
-# Command that moves one user in a voice channel into another voice channel specified by the user
 class MoveUsers(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+    
+    def find_similar_channel(self, channel_name, voice_channels):
+        """Find a similar voice channel by name."""
+        similarity_scores = {
```

Comment:
Out of curiosity, how do the similarty_scores look? IE: If I inputted "Rocket League" and we have "Rocket 🦵ue" in Konoha, what is the similarity score like?

Reply:
Rocket 🦵ue would be at the top of the list if there is no voice channel with the name "Rocket League". However I found some flaws with this function so I am still determining a better way to be able to find similar words

Reply:
Do you have any sample numbers? IE: The similarity score would be a 0.8 out of a scale of 0-1.